src/model/application.model.ts

```diff
@@ -10,7 +10,7 @@ All the field names in the page models must exactly match the `name` attributes
 */
 
 export interface ApplicationData {
-  add_member?: AddMember[]
+  member?: Member[]
```

Comment:
Can be changed to `members` please as it is a collection of objects. I will update `collaborator` key in a separate PR including a few naming convention.